cmd/graph.go

```diff
@@ -23,22 +23,67 @@ import (
 	"github.com/spf13/cobra"
 )
 
+var dep string
+
 var graphCmd = &cobra.Command{
 	Use:   "graph",
 	Short: "Generate a .dot file to be used with Graphviz's dot command.",
 	Long: `A graph.dot file will be generated which can be used with Graphviz's dot command.
 	For example to generate a svg image use:
 	twopi -Tsvg -o dag.svg graph.dot`,
 	RunE: func(cmd *cobra.Command, args []string) error {
-		depGraph, deps, _ := getDepInfo()
-		fileContents := "digraph {\ngraph [rankdir=TB, overlap=false];\n"
-		for _, dep := range deps {
-			_, ok := depGraph[dep]
-			if !ok {
-				continue
+		depGraph, deps, mainModule := getDepInfo()
+		// strict ensures that there is only one edge between two vertices
+		// overlap = false ensures the vertices don't overlap
+		fileContents := "strict digraph {\ngraph [overlap=false];\n"
+
+		// graph to be generated is based around input dep
+		if dep != "" {
```

Comment:
I think we should also add some tests, I see that this whole function does not have any...

Reply:
maybe let's do a function that received depGraph as parameter, and then we can assert output of both branches of the if here. WDYT?

Reply:
> I'm currently refactoring the code in this file into separate functions

Yup was doing that only :)